### PR TITLE
How to create an index.html at root level via getStaticPaths()

### PR DESCRIPTION
**Pull request by jablonski**
This is an update for the following conversation: https://github.com/withastro/docs/pull/1256

Couldn't update my initial PR, because I've deleted my fork.

### CODE REVIEW

File: src/pages/en/core-concepts/routing.md [hunk 1]
```diff
@@ -142,6 +142,53 @@ In this example, a request for `/withastro/astro/tree/main/docs/public/favicon.s
...
+```astro
+---
+// /src/pages/[...slug].astro
+export async function getStaticPaths() {
+  const pages = [
+    {
+      slug: undefined,
+      title: "Astro Store",
+      text: "Welcome to the Astro store!",
+    },
```
Comment (sarah11918) [inline src/pages/en/core-concepts/routing.md:158]:
```suggestion
#### Example: Dynamically create a top level root page

To create an index.html at root level (e.g. for content fetched from a headless CMS), include an object with `slug: undefined` in your `#getStaticPaths()` function.

```astro title="src/pages/[...slug].astro" "slug: undefined"
---
export async function getStaticPaths() {
  const pages = [
    {
      slug: undefined,
      title: "Astro Store",
      text: "Welcome to the Astro store!",
    },
```
This looks great @jablonski, thanks so much! I've made some minor adjustments to just tighten up the wording a bit and to add some code highlighting. Can you take a look and verify that it still expresses what you were going for?

Comment (jablonski) [reply src/pages/en/core-concepts/routing.md:158]: Perfect, thank you very much for improving my wording.

File: src/pages/en/core-concepts/routing.md [hunk 2]
```diff
@@ -142,6 +142,52 @@ In this example, a request for `/withastro/astro/tree/main/docs/public/favicon.s
...
+    return {
+      params: { slug },
+      props: { title, text },
+    };
+  });
+}
+
+const { title, text } = Astro.props;
+---
+
```
Comment (sarah11918) [inline src/pages/en/core-concepts/routing.md:179]:
```suggestion
```

**Comment by sarah11918** [comment]
Looks great, and thanks for working through this problem first with dev, then with docs, @jablonski! Welcome to Team Docs officially as a first-time contributor to the repository!  🥳

**Pull request merged**